src/gameplay/game.ts

```diff
@@ -1322,37 +1338,41 @@ class Game extends Room {
       whoAssassinShot2: this.whoAssassinShot2,
     };
 
-   /*  GameRecord.create(objectToStore, (err) => {
+    GameRecord.create(objectToStore, (err) => {
       if (err) {
         console.log(err);
       } else {
         console.log('Stored game data successfully.');
       }
-    }); */
-    
-    // Now recording games into dailyGameRecord instead of GameRecord. 
-    // The intended workflow is games -> dailyGameRecord -> (at end of rating preiod) Ratings Calculator -> archive into gameRecord
-    DailyGameRecord.create(objectToStore, (err) => {
+    });
+
+    //FR2 - Rating system 1 - record games for rating updates   
+    // This data will be used to calculate ratings at end of rating period.
+    // The intended workflow is games -> gameRatingRecord -> (at end of rating preiod) Ratings Calculator
+    const ratingRecordToStore = {
+      timeGameStarted: this.startGameTime,
+      timeGameFinished: timeFinished,
+      winningTeam: this.winner,
+      spyTeam: this.spyUsernames,
+      resistanceTeam: this.resistanceUsernames,
+      numberOfPlayers: this.playersInGame.length,
+      roomCreationType: this.roomCreationType,
```

Comment:
Do we need this? Can we simply surround this in an if statement?

`if this.roomCreationType === roomCreationType.RankedQueue`?

Reply:
Yeah, that makes sense, I'll get on that

Reply:
I've wrapped the whole "recording ranked games" section into an if-block that checks if the current game is a ranked games.